app/controllers/tenants/projects_controller.rb

```diff
@@ -6,7 +6,9 @@ def index
     redirect_to root_path if @tenant.nil?
 
     if params[:category_name] == "Shop All" || params[:category_name].nil?
-      @projects = Project.includes(:photos).active
+      @projects = Project.paginate(page: params[:page],
+                                   per_page: 10).
+                                  includes(:photos).active
```

Comment:
In Atom, I see both spaces and tabs here, and they don't look aligned...try to make sure that you can see hidden characters in your editor.